app/views/admin/feature_flags/index.html.erb

```diff
@@ -0,0 +1 @@
+<%= render Admin::FeatureFlags::FeatureFlagsListComponent.new(feature_flags: @feature_flags) %>
```

Comment:
Tieto oneliner templates neuznavam uplne. Komponentov sa uz snazime trosku zbavit pokial to nie su ozajstne komponenty (teda nepouzivaju sa naozaj 2-3x niekde inde). cize tuto sprav v klude inline toho html co je v tom componente